src/ResourceManager/AzureBatch/Commands.Batch/help/Get-AzureBatchJobPreparationAndReleaseTaskStatus.md

```diff
@@ -0,0 +1,180 @@
+---
+external help file: Microsoft.Azure.Commands.Batch.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureBatchJobPreparationAndReleaseTaskStatus
+
+## SYNOPSIS
+Gets Batch job preparation and release task status.
+
+## SYNTAX
+
+### Id (Default)
+```
+Get-AzureBatchJobPreparationAndReleaseTaskStatus [-Id] <String> [-Filter <String>] [-MaxCount <Int32>]
+ [-Select <String>] [-Expand <String>] -BatchContext <BatchAccountContext> [<CommonParameters>]
+```
+
+### InputObject
+```
+Get-AzureBatchJobPreparationAndReleaseTaskStatus [-Job] <PSCloudJob> [-Filter <String>] [-MaxCount <Int32>]
+ [-Select <String>] [-Expand <String>] -BatchContext <BatchAccountContext> [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzureBatchJobPreparationAndReleaseTaskStatus** cmdlet gets the Azure Batch job preparation and release task status for a Batch job.
+You must supply the *Id* parameter or a **PSCloudJob** instance to this cmdlet.
+
+## EXAMPLES
+
+### Example 1: Get the job preparation and release status of a job
```

Comment:
@matthchr do you think it be beneficial for users to have an additional example that used any of the filtering parameters?